docs/develop/03_workshop/05_claimer/index.md

```diff
@@ -6,11 +6,32 @@ title: 👤 Claimer
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-In this section you will walk though all processes done by the <span className="label-role claimer">Claimer</span>.
+This section covers the steps undertaken by the <span className="label-role claimer">Claimer</span>.
+
+Here's an overview:
+
+1. [Create a DID](./01_did.md), which is the identity used to interact with <span className="label-role attester">Attesters</span> and <span className="label-role verifier">Verifiers</span>.
+2. Create a claim, request an attestation, and generate a credential using the attestation for our claim.
+3. Present the claim to a <span className="label-role verifier">Verifier</span>.
+
+## What is a Claimer?
+
+A Claimer is an individual or institution that makes a claim or statement about their identity or abilities.
+They can use their identity credentials to prove these claims, and third-party institutions verify them.
+
+Anyone can be a Claimer.
+All you need to do is complete a Ctype and create a claim.
+Then, you can send these claims to Attesters for verification.
+
+Claimers are crucial in the Self-Sovereign Identity system.
```

Comment:
Not sure if this is necessary to say or maybe it needs to be rephrased. 

Maybe `claimers are first class citizen within self-sovereign identity systems.`